src/sass/components/_hero-header.scss

```diff
@@ -0,0 +1,67 @@
+.container--header-bg {
+  background-color: $pinkBg;
+}
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 6px;
+  &__hamburger {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+  }
+}
+.container {
+  overflow: clip;
+  position: relative;
```

Comment:
Jeżeli chodzi o te elementy jak koło i zdjęcie loda, one muszą być absolute, a więc relative musi być ich rodzic. Skoro na li ani na ul nie masz klasy to najbliższym rodzicem  jest `<section>` i tam zrób `position: relative;`

Reply:
<div class="hero"> ma aktualnie position:relative.